### PR TITLE
update sier

### PR DESCRIPTION
**Pull request by dleng16**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -103,4 +103,13 @@ dist
 # TernJS port file
 .tern-port
 
+#python virtual environment
```
Comment (jchen006) [inline .gitignore:106]: Is this the only stuff? I thought there's like other things like `.DS_store`and what not? Make sure you do a run through to verify.

File: data_models/SIR.py [hunk 2]
```diff
@@ -7,13 +7,32 @@
...
+    """
+
     def default(self, obj):
         if isinstance(obj, np.ndarray):
             return obj.tolist()
         return JSONEncoder.default(self, obj)
 
 
 def deriv(y, time, N, beta, gamma, delta):
+    """
```
Comment (jchen006) [inline data_models/SIR.py:22]: 👍  love these comments and very well done!

Comment (dleng16) [reply data_models/SIR.py:22]: thank u

File: .gitignore (hunk 1 above)
Comment (dleng16) [reply .gitignore:106]: ok

Comment (jchen006) [reply .gitignore:106]: are you sure you covered everything here? https://github.com/github/gitignore/blob/master/Python.gitignore @dleng16

**Comment by jchen006** [comment]
@djleng dont' forget to delete the .DS_store

**Pull request merged**